test/promote_file.md.corrected

```diff
@@ -0,0 +1,64 @@
+Mdx can also understand ocaml code blocks:
```

Comment:
Other tests use the file extension `.expected` (e.g. that's expected that the generated `.corrected` file is similar to that file).